app/lib/coursesHelper.coffee

```diff
@@ -224,6 +236,8 @@ module.exports =
   courseLabelsArray: (courses) ->
     courses.map((course) -> course.acronym())
 
+  hasUserCompletedCourse: hasUserCompletedCourse
```

Comment:
I did it because I didn't want the function as part of the module.exports method, which essentially marks everything as public by default.

I feel module.exports is more explicit when it's just a list of exports instead of the code included. I'm happy to change it though.

Reply:
I feel the impact is greater when you're composing private functions to make a useful public functions which you then export. Current code seems to have everything in an object or class that is all exported. Even though there is no private function here, this style would allow this function to then be composed with the other public functions that are exported.

For example, the looping in this file is very similar between many functions. Extracting that out of the module.export makes it easier to reuse.